### PR TITLE
[ENGAGE-648] - Add missing translations

### PR DESCRIPTION
**Pull request by marihemori**
### Type of Change

* * [x] Bugfix
* * [ ] Feature
* * [ ] Code style update (formatting, local variables)
* * [ ] Refactoring (no functional changes, no api changes)
* * [ ] Tests
* * [ ] Other

### Motivation and Context
Some word translations were missing.

### Summary of Changes
- Added translations for some words

### CODE REVIEW

File: src/components/settings/lists/ListSectorQueues.vue [hunk 1]
```diff
@@ -45,25 +45,28 @@ export default {
...
+          id: 'agents',
+          text: this.$t('queues.quantity_agents'),
+          flex: 3,
+        },
+        {
+          id: 'visualize',
+          text: this.$t('queues.visualize'),
+          flex: 2,
+        },
+        // outros cabeçalhos de tabela...
```
Comment (Aldemylla) [inline src/components/settings/lists/ListSectorQueues.vue:66]: Was this comment forgotten here?

File: src/locales/translations.json [hunk 2]
```diff
@@ -207,6 +207,41 @@
...
+      "pt-br": "Nome do setor",
+      "en": "Sector name",
+      "es": "Nombre del sector"
+    },
+    "placeholder": {
+      "pt-br": "Exemplo: Financeiro",
+      "en": "Example: Financial",
+      "es": "Ejemplo: Financiero"
+    },
+    "save": {
```
Comment (Aldemylla) [inline src/locales/translations.json:240]:
```suggestion
    "save_and_continue": {
```

File: src/locales/translations.json [hunk 3]
```diff
@@ -301,6 +336,46 @@
...
+    "queue_edit": {
+      "pt-br": "Editar",
+      "en": "Edit",
+      "es": "Editar"
+    },
+    "queue_table_name": {
+      "pt-br": "Nome",
+      "en": "Name",
+      "es": "Nombre"
+    },
```
Comment (Aldemylla) [inline src/locales/translations.json:369]: Translations with just one word can be exposed outside the `sector` object, to facilitate future use. In fact, `edit` is already available in this file, so there is no need to repeat it here.

File: src/locales/translations.json [hunk 4]
```diff
@@ -301,6 +336,46 @@
...
+    "quantity_agents": {
+      "pt-br": "Qtd de agentes",
+      "en": "Number of agents",
+      "es": "Cantidad de agentes"
+    },
+    "visualize": {
+      "pt-br": "Visualizar",
+      "en": "Visualize",
+      "es": "Para ver"
```
Comment (Aldemylla) [inline src/locales/translations.json:378]: This traduction is already available as `view` in this file, so there is no need to repeat it here.

File: src/locales/translations.json [hunk 5]
```diff
@@ -858,6 +933,11 @@
         "pt-br": "Exemplo: Dúvidas",
         "en": "Example: Doubts",
         "es": "Ejemplo: Dudas"
+      },
+      "tag": {
+        "pt-br": "Adicionar",
+        "en": "Add",
+        "es": "Agregar"
```
Comment (Aldemylla) [inline src/locales/translations.json:940]: This traduction is already available as `add` in this file, so there is no need to repeat it here.

File: src/components/settings/lists/ListSectorQueues.vue (hunk 1 above)
Comment (marihemori) [reply src/components/settings/lists/ListSectorQueues.vue]: I forgot this comment!

**Review by Aldemylla — APPROVED** [review]

**Pull request merged**